doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -962,8 +946,9 @@ This operation supports two options:
 </td></tr>
 </table>
 
-Returns a list of entity types, either as JSON object (including the union set of attribute names along all the entities in each type) or
-a JSON array (just type name), depending on the `collapsed` option.
+Returns a JSON array of existing entity types. The elements of the array are either JSON objects (including
```

Comment:
Returns a JSON array containing one entry per entity type present in the system. 

Array elements are always objects, aren't them?